examples/semi_supervised/plot_label_propagation_structure.py

```diff
@@ -15,11 +15,14 @@
 #          Andreas Mueller <amueller@ais.uni-bonn.de>
 # License: BSD
 
+# %%
```

Comment:
Let's improve this part. Instead, to create a block with all imports, we can move each import in the cell where it is used first.